src/renderer/viz/expressions/constants.js

```diff
@@ -7,3 +7,4 @@ export const DEFAULT_OPTIONS = {
     othersLabel: OTHERS_LABEL,
     samples: DEFAULT_SAMPLES
 };
+export const MIN_SAFE_INTEGER = Math.pow(-2, 24);
```

Comment:
(-2)^24 is positive, not negative. One way to do this is `-(1<<24)`

Also, I'd change the naming to `FP32_MIN_SAFE_INTEGER`, because with this people may think its the FP64 one